app/konnect/deployment/import.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Import Kong Gateway Entities into Konnect Cloud
+no_version: true
+---
+
+You can attach any edition of self-managed {{site.base_gateway}} to
+{{site.konnect_saas}} as a data plane, and use Konnect as your control plane.
+
+Use [decK](/deck/) to import Kong Gateway entity configuration into a runtime
+group in your {{site.konnect_short_name}} organization.
```

Comment:
Suggestion: Follow up to my last comment on "attach", this could be the actual intro:

```
If you are an existing Kong Gateway user looking to use Konnect as your cloud-hosted control plane. You can use [decK](/deck/) to import your Kong Gateway entity configuration into a runtime
group in your {{site.konnect_short_name}} organization.
```